<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="2" parallel="methods" name="Failed suite [Suite1]" guice-stage="DEVELOPMENT">
  <test thread-count="2" parallel="methods" name="Test2(failed)">
    <classes>
      <class name="com.automationpractice.testcases.Tc_002_Login_With_valid_Credentials">
        <methods>
          <include name="launchBrowser"/>
          <include name="teardown"/>
          <include name="Verify_Login_With_Valid_Credentials"/>
          <include name="beforeSuite"/>
        </methods>
      </class> <!-- com.automationpractice.testcases.Tc_002_Login_With_valid_Credentials -->
      <class name="com.automationpractice.testcases.TC_001_Create_New_Account">
        <methods>
          <include name="Create_New_Account"/>
          <include name="launchBrowser"/>
          <include name="teardown"/>
          <include name="beforeSuite"/>
        </methods>
      </class> <!-- com.automationpractice.testcases.TC_001_Create_New_Account -->
    </classes>
  </test> <!-- Test2(failed) -->
</suite> <!-- Failed suite [Suite1] -->
